scripts/ots-app.js

```diff
@@ -20,6 +20,17 @@ jQuery.noConflict();
           }
         });
     },
+
+    dropdownOpen : function() {
+      $('.dropdown').on('click', 'a', function(event){
+        var target = event.target
+        if($(target).parent().hasClass('open')){
```

Comment:
Instead of checking the class state yourself, how about using the [`.toggleClass(className)`](http://api.jquery.com/toggleclass) method of jQuery? It might be easier to read and is also results in less code to maintain 😄